packages/vector_graphics_codec/pubspec.yaml

```diff
@@ -1,10 +1,14 @@
 name: vector_graphics_codec
 description: A simple command-line application.
-version: 0.0.0
+version: 0.0.1
 homepage: https://github.com/dnfield/vector_graphics
 
 environment:
   sdk: '>=2.12.0 <3.0.0'
 
+dependencies:
+  typed_data: ^1.3.0
```

Comment:
Can we avoid depending on this? It's a vector for version mismatching and I'm not sure how much it really gets us.

Reply:
Done